### PR TITLE
Create endpoint and environment variable to configure the log level

### PR DESCRIPTION
**Pull request by kevinuehara**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

File: DeviceManager/LoggerHandler.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            'level': LOG.get_log_level()
+        }
+
+        return result
+
+
+@logger.route('/log', methods=['PUT'])
+def flask_update_log_level():
+    try:
+        result = LoggerHandler.update_log_level(request)
```
Comment (raulnegreiros) [inline DeviceManager/LoggerHandler.py]: Isn't a good practice use the request object into the handler, the handler must receive only the info necessary to realize its work, and not an HTTP request

Comment (raulnegreiros) [inline DeviceManager/LoggerHandler.py]: If the result is always a success, why do I need a result?

File: README.md [hunk 2]
```diff
@@ -33,23 +33,24 @@ But you won't need to worry about installing any of these - they are automatical
...
+KAFKA_HOST              | kafka host                                                    | kafka
+KAFKA_PORT              | kafka port                                                    | 9092
+STATUS_TIMEOUT          | kafka timeout                                                 | 5
+BROKER                  | kafka topic subject manager                                   | http://data-broker
+REDIS_HOST              | redis host                                                    | device-manager-redis
+REDIS_PORT              | redis port                                                    | 6379
+DEV_MNGR_CRYPTO_PASS    | password of crypto                                            | none
+DEV_MNGR_CRYPTO_IV      | inicialization vector of crypto                               | none
+DEV_MNGR_CRYPTO_SALT    | salt of crypto                                                | none
+LOG_LEVEL               | logger level (DEBUG, ERROR, WARNING, CRITICAL, INFO, NOTSET)  | INFO
```
Comment (raulnegreiros) [inline README.md]: documentation update! :1st_place_medal:

File: DeviceManager/Logger.py [hunk 3]
```diff
@@ -27,7 +31,30 @@ def __init__(self, LOG_LEVEL = logging.DEBUG,
             self.log.setLevel(LOG_LEVEL)
             self.log.addHandler(self.stream)
             self.log.handler_set = True
+
+    def update_log_level(self, LEVEL):
+        levelToName = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']
```
Comment (raulnegreiros) [inline DeviceManager/Logger.py]: what means 'notset'?

Comment (kevinuehara) [reply DeviceManager/Logger.py]: Notset is an option available configuration log that disable it. The new PR removes this option.

File: DeviceManager/LoggerHandler.py (hunk 1 above)
Comment (kevinuehara) [reply DeviceManager/LoggerHandler.py]: I reviewed and doesn't really make sense with the purpose. Initially, I thinked that the result would be manipulated. The new PR, removes this variable and just return if the request was a success or not.

Comment (kevinuehara) [reply DeviceManager/LoggerHandler.py]: I updated in the new PR. Thanks!

File: DeviceManager/LoggerHandler.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            'level': LOG.get_log_level()
+        }
+
+        return result
+
+
+@logger.route('/log', methods=['PUT'])
+def flask_update_log_level():
+    try:
+        _, json_payload = parse_payload(request, log_schema)
```
Comment (raulnegreiros) [inline DeviceManager/LoggerHandler.py]: for the sake of curiosity,  if the HTTP request contains an invalid JSON it throws an HTTPRequestError?

Comment (kevinuehara) [reply DeviceManager/LoggerHandler.py]: Yes, the method called parse_payload validate the JSON format of request and also the fields passed like argument in schema, returning a HTTPRequestError.

Comment (raulnegreiros) [reply DeviceManager/LoggerHandler.py]: Nice

**Pull request merged**